docs/units-and-global-variables.rst

```diff
@@ -167,6 +167,10 @@ For more information, see the section on :ref:`address`.
     to make safe Ether transfers, always check the return value of ``send``, use ``transfer`` or even better:
     Use a pattern where the recipient withdraws the money.
 
+.. note::
+   If storage variables are accessed via a low-level delegatecall, the storage locations of the two contracts
+   must align in order for the called contract to write to the storage of the calling contract.
```

Comment:
I wouldn't say "to write to the storage" because it suggest that writing would fail on misalignment, which isn't the case. Just it won't be possible to interpret the storage as intended.

Probably "to write to the correct storage locations of the calling contract." is clearer

Reply:
While we are at it: of course it applies to reads as well as writes :)